### PR TITLE
Donuts index

### PR DESCRIPTION
**Pull request by eschwelling**
donuts index page and feature test.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,17 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+Shop.create!(name: "Dunkin' Donuts", address: "56 Westview Lane", city: "Pittsburgh", state: "PA", url:"www.google.com", phone:"123-456-7890")
```
Comment (bkincart) [inline db/seeds.rb]: Please fix the indentation on this line 8!

File: spec/features/user_visits_donut_index_page_spec.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+feature 'user visits donut index page' do
+  scenario 'user sees a list of donuts' do
+
```
Comment (bkincart) [inline spec/features/user_visits_donut_index_page_spec.rb]: remove whitespace

File: db/seeds.rb (hunk 1 above)
Comment (eschwelling) [reply db/seeds.rb]: done!

File: spec/features/user_visits_donut_index_page_spec.rb (hunk 2 above)
Comment (eschwelling) [reply spec/features/user_visits_donut_index_page_spec.rb]: donezo!

**Pull request merged**